app/api/chat/route.ts

```diff
@@ -3,78 +3,178 @@ import { z, ZodError } from "zod";
 import formatZodError from "@/utils/format-zod-error";
 import { ChatSessionsTable, InboxesTable, ContactsTable } from "@/db/schema";
 import { eq } from "drizzle-orm";
-import { NotFoundError } from "@/server/errors";
 import { getDb } from "@/db/connection";
+import { createErrorResponse } from "@/utils/error-response";
 
 export async function POST(request: Request) {
+  const requestId = `chat-session-${Date.now()}`;
+  console.log(`[${requestId}] Creating new chat session`);
+  
   try {
-    const body = await request.json();
```

Comment:
Why did we add all of these changes to every route? We should only update the test slack route